projects/project-tooling.md

```diff
@@ -0,0 +1,38 @@
+# Project Tooling SIG
```

Comment:
What about "Infrastructure SIG", similar to what [k8s has](https://github.com/kubernetes/community/blob/master/sig-k8s-infra/README.md), or is this too big of a scope what you are thinking of?

Reply:
Could that be confused with a potential SIG related to Infrastructure instrumentation or conventions?

Reply:
How about 'Project Infrastructure'?